content/docs/1.2.0/agents/cli-usage.md

```diff
@@ -0,0 +1,199 @@
+# Agent CLI Usage
+
+```sh
+iofog-agent command --option <argument>
+```
+
+## Commands
+
+|                             |                                                                             |
+| --------------------------- | --------------------------------------------------------------------------- |
```

Comment:
I would remove the period to match formatting for the rest of the descriptions in tables.